micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -65,13 +68,15 @@ public MnProcessEngineConfiguration(SynchronousTransactionManager<Connection> tr
                                         DataSource dataSource,
                                         MnArtifactFactory artifactFactory,
                                         MnBeansResolverFactory beansResolverFactory,
-                                        ProcessEngineConfigurationCustomizer processEngineConfigurationCustomizer) {
+                                        ProcessEngineConfigurationCustomizer processEngineConfigurationCustomizer,
+        BasicJdbcConfiguration basicJdbcConfiguration) {
         this.transactionManager = transactionManager;
         this.jobExecutor = jobExecutor;
         this.telemetryRegistry = telemetryRegistry;
         this.beansResolverFactory = beansResolverFactory;
         this.environment = environment;
         this.camundaBpmVersion = camundaBpmVersion;
+        this.basicJdbcConfiguration = basicJdbcConfiguration;
```

Comment:
please also move this before the datasource

Reply:
it is before setDatasource(...)

```
        this.basicJdbcConfiguration = basicJdbcConfiguration;
        setDataSource(dataSource);
```